simulation.py

```diff
@@ -0,0 +1,84 @@
+"""
+Fuctions for a process to simulate an alpha.
+"""
+
+import copy
+import importlib
+import sys
+from thousandaire.constants import DATA_LIST_ALL
+from thousandaire.constants import TRADING_CONFIG, TRADING_REGION
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+PRICE_DATASET = 'price_dataset'
+
+def initialize():
+    """
+    Initialize data we get.
+    """
+    raw_data = DataLoader(DATA_LIST_ALL).get_all()
+    workdays = {
+        region : raw_data['workdays'][region]
+        for region in TRADING_REGION if region in raw_data['workdays'].keys()}
+    return_data = {}
+    for key, value in workdays.items():
```

Comment:
Can we use meaningful terms instead of `key, value`? For example, `region, workdays` should be better, with current `workdays` renamed to be `workdays_all`.